sdks/python/apache_beam/transforms/core.py

```diff
@@ -2270,3 +2286,163 @@ def to_runner_api_parameter(self, unused_context):
   @PTransform.register_urn(common_urns.primitives.IMPULSE.urn, None)
   def from_runner_api_parameter(unused_parameter, unused_context):
     return Impulse()
+
+
+class ApproximateUniqueGlobally(PTransform):
```

Comment:
Since the file is growing, is it better to separate each transform to it's own file? I am happy to make that change.

Reply:
That make sense. Also probably core.py is not the right place anyway.

Reply:
Does stats.py sound good?